sematic/resolvers/tests/test_log_streamer.py

```diff
@@ -86,7 +86,7 @@ def test_ingested_logs_uncaught():
         upload_number += 1
```

Comment:
Why not just look at the last file instead of concatenating the contents of all previous files, which contain incomplete duplicated of the same info?

Reply:
it's helpful to be able to confirm that the contents are properly cumulative. That doesn't matter so much for this particular test, but it does for others in the file.